code/modules/uplink/uplink_items/ammunition.dm

```diff
@@ -59,6 +59,16 @@
 	cost = 2
 	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
 
+/datum/uplink_item/ammo/deagle
+	name = ".50 AE Magazine"
+	desc = "An additional 7-round .50 AE magazine, compatible with the Desert Eagle pistol. \
+			Just in case you need to turn your enemies OFF."
```

Comment:
```suggestion
		Just in case you need to turn your enemies OFF."
```

no tab alignment

Reply:
Done, or should be!

Reply:
Oh whoops, I pulled it back too many tab aligns. Just a second.

Reply:
Wait, the .357 speedloader has the same amount of indention the .50 AE mag does. Should I keep it as is (how it is now) or re-add an indent?

Reply:
Up to you, we don't require changing old code but it's correct to fix it

Reply:
I'll knock all the unneeded indents out and then resolve this when I have. Thanks!

Reply:
Should be done! Let me know if it isn't.